lib/beaker/host/unix/exec.rb

```diff
@@ -233,15 +233,8 @@ def ssh_permit_user_environment
   def environment_string env
     return '' if env.empty?
     env_array = self.environment_variable_string_pair_array( env )
-
     environment_string = env_array.join(' ')
-    command = 'env'
-    punctuation = ''
-    if self[:platform] =~ /cisco-5/
-      command = 'export'
-      punctuation = ';'
-    end
-    "#{command} #{environment_string}#{punctuation}"
+    "env #{environment_string}"
```

Comment:
in the refactor of the cisco code, we made a copy of the `environment_string` method for cisco.  You'll see that the logic has been moved there, and should be able to check that Cisco hosts inherit from Unix hosts correct, are instantiated in `host.rb` as needed for this to work.